I get an error while I run "mvn test" on the project

For anything other than bug reports and feature requests (performance, refactoring, etc),
just go ahead and file the issue. Please provide as many details as possible.

If you have a question or a support request, please open a new discussion on [GitHub Discussions](https://github.com/modelcontextprotocol/java-sdk/discussions)

Please do **not** create issues on the [Issue Tracker](https://github.com/modelcontextprotocol/java-sdk/issues) for questions or support requests.
We would like to keep the issue tracker **exclusively** for bug reports and feature requests.


The github discussions link gives a 404. So posting it here.
I get this error when I run `mvn test` on the project. 
I have Docker CLI installed. What is the fix here?

```
[ERROR] Tests run: 20, Failures: 0, Errors: 20, Skipped: 0, Time elapsed: 0.089 s <<< FAILURE! -- in io.modelcontextprotocol.client.ServletSseMcpSyncClientTests
[ERROR] io.modelcontextprotocol.client.ServletSseMcpSyncClientTests.testInitializeWithRootsListProviders -- Time elapsed: 0.013 s <<< ERROR!
java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:229)
	at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:154)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:196)
	at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:108)
	at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:109)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:321)
	at io.modelcontextprotocol.client.ServletSseMcpSyncClientTests.onStart(ServletSseMcpSyncClientTests.java:37)
	at io.modelcontextprotocol.client.AbstractMcpSyncClientTests.setUp(AbstractMcpSyncClientTests.java:59)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
```

--- a/README.md
+++ b/README.md
@@ -17,13 +17,15 @@ For comprehensive guides and SDK API documentation, visit the [MCP Java SDK Refe
 ### Building from Source
 
 ```bash
-mvn clean install
+./mvnw clean install -DskipTests
 ```
 
 ### Running Tests
 
+To run the tests you have to pre-install `Docker` and `npx`.
+
 ```bash
-mvn test
+./mvnw test
 ```
 
 ## Contributing

